extensions/roc-plugin-test-jest/src/config/roc.config.js

```diff
@@ -0,0 +1,14 @@
+export default {
+    settings: {
+        test: {
+            jest: {
+                config: {
+                },
+                reporter: {
```

Comment:
You mean that we should have a object that control the report format for all of the test framworks ? It can be tricky to get it to work for all plugins.